modules/claims_api/app/controllers/claims_api/v2/veterans/intent_to_file_controller.rb

```diff
@@ -16,10 +16,12 @@ def type
           validate_request!(ClaimsApi::V2::ParamsValidation::IntentToFile)
 
           type = get_bgs_type(params)
-          response = bgs_service.intent_to_file.find_intent_to_file_by_ptcpnt_id_itf_type_cd(
+          response = local_bgs_service.find_intent_to_file_by_ptcpnt_id_itf_type_cd(
             target_veteran.participant_id,
             type
           )
+          message = "No active '#{params[:type]}' intent to file found."
```

Comment:
Nitpick: you can have some fun with this. 
```
"detail": "No active 'SurViVoR' intent to file found."
```
I'd say either `.downcase` or `.capitalize` it.

Reply:
I'm not as worried about these being in an `if` either (like mentioned above), as it's just using the `params[:type]`.. probably not as concerning to bypass.